docs/design/NonContainerizedNSFSDesign.md

```diff
@@ -15,69 +15,74 @@ node src/cmd/nsfs ../standalon/nsfs_root --config_dir ../standalon/fs_config
 
 ### 1. Accounts
 
-- Account will have a directory and in that, it will map 1 file per account, and the file name will be {access_key}.json
-- Secrets will be saved either as plain text or Encrypted strings? [TBD]
-- JSON Schema - $ref: account_api#/definitions/account_info
-- new_buckets_path in the account schema will be used to populate the namespace store(read_resources, write_resource).
-- BucketSpace interface is updated with the new method `read_account_by_access_key()`
+- Account will have a directory and in that, it will map 1 file per account, and the file name will be {account_name}.json
+- Secrets will be saved as plain text
+  - Note: We had the option to save it as plain text or Encrypted strings.
+- JSON Schema - `src/server/object_services/schemas/nsfs_account_schema.js`.
+  - Note: It is based on $ref: account_api#/definitions/account_info.
+- `new_buckets_path` in the account schema will be used to populate the namespace store (read_resources, write_resource).
+- `BucketSpace` interface is updated with the new method `read_account_by_access_key()`
     - Method will read multiple account config files referring to access_key and return the account to `account_cache`.
 
-```
+```json
 {
 	"name": "user1",
 	"email": "user1@noobaa.io",
-	"has_login": "false",
-	"has_s3_access": "true",
-    "allow_bucket_creation": "true",
-	"access_keys": [{
-		"access_key": "aa-abcdefghijklmn123456",
-		"secret_key": "ss-abcdefghijklmn123456"
-	}],
+  "creation_date": "2024-01-11T08:24:14.937Z",
+	"access_keys": [
+    {
+		  "access_key": "aa-abcdefghijklmn123456",
```

Comment:
please change these to valid access key/secret key

Reply:
Added a valid secret_key, but I have to set the suffix of EXAMPLE, please see the comment.